src/Foundatio.Extensions.Hosting/Startup/StartupActionsContext.cs

```diff
@@ -28,27 +28,27 @@ internal void MarkStartupComplete(RunStartupActionsResult result)
     public async Task<RunStartupActionsResult> WaitForStartupAsync(CancellationToken cancellationToken, TimeSpan? maxTimeToWait = null)
     {
         bool isFirstWaiter = Interlocked.Increment(ref _waitCount) == 1;
-        var startTime = SystemClock.UtcNow;
-        var lastStatus = SystemClock.UtcNow;
+        var startTime = DateTime.UtcNow;
```

Comment:
Still think this needs to use system clock as this could be abstracted during tests?